lab-sarah/README.md

```diff
@@ -0,0 +1,59 @@
+# Project description
+
+Creates a RESTful multiple-resource API with a persistence layer powered by Mongo. Uses CRUD operations to create, update, and destroy 'Dogs' and 'Parks'. Utilizes Express as the server-side framework.
+
+## How to install
+
+See package.json for list of dependencies necessary to install and run this project.
+
+## How to start the server
+
+From the root level of the project, enter 'npm start' in terminal to start the server.
+
+## Server endpoints
+
+### Dog Routes
+
+POST requests
+
+/api/park/:parkID/dog
+
+* Pass a new 'dog' as valid JSON into the body of the request to create a dog, and pass a parkID into the query string to specify the park the dog should reside in.
+
+GET requests
+
+/api/park/dog/:dogID
+
+* Pass a dog id to fetch a specific dog. No parkID required in query string.
+
+
+DELETE requests
+
+/api/dog/:dogID
+
+* Pass a dog id to delete that dog. No parkID required in query string.
+
+
+### Park routes
+
+POST requests
+
+/api/park
+
+* Pass in a new 'park' as valid JSON into the body of the request to create a park.
+
+GET requests
+
+/api/park/:id
+
+* Pass in a park id to fetch that park.
+
+DELETE requests
+
+/api/park/:id
+
+* Pass in a park id to delete that park.
+
+PUT requests
+
+* Pass in a park id and valid JSON into the body of the request to update a park.
```

Comment:
Your docs are very well-written, but be sure to let the user know what properties must be included when they POST a dog or a park, otherwise they won't be successful :)